docs/advanced/peering/peering-via-cr.md

```diff
@@ -0,0 +1,428 @@
+    liqo.io/remote-cluster-id: <PROVIDER_CLUSTER_ID>
+  annotations:
+    liqo.io/remote-tenant-namespace: <PROVIDER_TENANT_NAMESPACE>
+  name: cplane-secret
+  namespace: <TENANT_NAMESPACE>
+```
+
+Once you create this secret, the `liqo-crd-replicator` starts the replication of the resources, and enables the creation of `ResourceSlice` resources targetting the provider cluster. This allows the consumer to start negotiating resources with the provider cluster.
+
+After the secret creation, the logs of the `liqo-crd-replicator` should contain the following records:
+
+```text
+ k logs -n liqo liqo-crd-replicator-68f9f55dfc-bc4l8 --tail 5
+
+I1107 10:05:29.249872       1 crdReplicator-operator.go:94] Processing Secret "liqo-tenant-cl02/kubeconfig-controlplane-cl02"
+I1107 10:05:29.254977       1 reflector.go:90] [cl02] Starting reflection towards remote cluster
+I1107 10:05:29.255001       1 reflector.go:131] [cl02] Starting reflection of offloading.liqo.io/v1beta1, Resource=namespacemaps
+I1107 10:05:29.255035       1 reflector.go:131] [cl02] Starting reflection of authentication.liqo.io/v1beta1, Resource=resourceslices
+I1107 10:05:29.355741       1 reflector.go:163] [cl02] Reflection of authentication.liqo.io/v1beta1, Resource=resourceslices correctly started
+```
+
+### Summary of authentication configuration
+
+To sum up, to set up the authentication, **on the cluster provider** you will need to:
+
+- Create the credentials to be used by the consumer
+- Bind the credentials to the `liqo-remote-controlplane` role
+- Create a `Tenant` resource for the consumer
+
+While, **on the consumer side**:
+
+- Create a new secret containing the kubeconfig with the credentials to access to the cluster provider
+
+## Declarative configuration of namespace offloading
+
+While offloading is independent from the network, which means that it is possible to negotiate resources and configure a namespace offloading without the inter-cluster network enabled, **a [working authentication configuration](#declarative-configuration-of-clusters-authentication) is a pre-requisite to enable offloading**.
+
+### Ask for resources: configure a ResourceSlice
+
+The `ResourceSlice` resource is the one to be created to ask a cluster provider for resources.
+The following is an example of `ResourceSlice`:
+
+```{code} yaml
+apiVersion: authentication.liqo.io/v1beta1
+kind: ResourceSlice
+metadata:
+  annotations:
+    liqo.io/create-virtual-node: "true"
+  creationTimestamp: null
+  labels:
+    liqo.io/remote-cluster-id: <PROVIDER_CLUSTER_ID>
+    liqo.io/remoteID: <PROVIDER_CLUSTER_ID>
+  name: test
+  namespace: liqo-tenant-cl02
+spec:
+  class: default
+  providerClusterID: <PROVIDER_CLUSTER_ID>
+  resources:
+    cpu: 20
+    ram  128Gi
+```
+
+With the configuration above, once the resources are accepted from the provider side, a new (virtual) node, impersonating the provider cluster, will make available the requested resorces on the consumer cluster.
```

Comment:
```suggestion
The above request has to be accepted by the provider cluster.
If the request is successful, a new (virtual) node, impersonating the provider cluster, will make available the requested resources on the consumer cluster.
```

Reply:
I changed a little bit the order of sentences to address both this comment and the one above